RecoTracker/IterativeTracking/python/TobTecStep_cff.py

```diff
@@ -21,7 +21,7 @@
 tobTecStepSeedLayersTripl = cms.EDProducer("SeedingLayersEDProducer",
     layerList = cms.vstring(
     #TOB
-    'TOB1+TOB2+MTOB3',
+    'TOB1+TOB2+MTOB3','TOB1+TOB2+MTOB4',
```

Comment:
On 9/24/15 2:11 PM, Vincenzo Innocente wrote:
> In RecoTracker/IterativeTracking/python/TobTecStep_cff.py
> <https://github.com/cms-sw/cmssw/pull/11467#discussion_r40359813>:
> 
>> @@ -21,7 +21,7 @@
>>  tobTecStepSeedLayersTripl = cms.EDProducer("SeedingLayersEDProducer",
>>      layerList = cms.vstring(
>>      #TOB
>> -    'TOB1+TOB2+MTOB3',
>> +    'TOB1+TOB2+MTOB3','TOB1+TOB2+MTOB4',
> 
> how it when in here?
> I stashed before commit!
> will remove!
> still quite disappointed by git behaviour...
> 

ok removal from a typo or something is an easy solution for me : the
reco  signature need goes away.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/11467/files#r40359813>.
>